wasm/validator/src/permission.rs

```diff
@@ -82,14 +82,17 @@ pub mod asset {
     }
 
     impl PassCondition for Owner<'_> {
-        fn validate(&self, authority: &AccountId, _block_height: u64) -> Result {
-            if self.asset_id.account_id() != authority {
-                return Err(ValidationFail::NotPermitted(
-                    "Can't access asset owned by another account".to_owned(),
-                ));
+        fn validate(&self, authority: &AccountId, block_height: u64) -> Result {
+            if block_height == 0 {
```

Comment:
The fact that ```block_height == 0``` skips all other checks is a bit confusing but as I understand it was done on purpose.

Reply:
I will turn this comment into an issue so we can discuss better ways of doing this. The thing is that genesis account has to have the privilege to execute any ISI. Since we've introduced validation of genesis transactions in #3781 this was no longer the case and would not allow genesis to be applied